### DISCUSSION TITLE
i have a problem

### DISCUSSION THREAD

**Discussion by Hourunze1997** [FreshRSS/FreshRSS#7109] [Q&A]
I want to use the SharebyEmail extension in FreshRSS, but the page shows that this extension cannot be configured. I want to know if this extension is no longer available.

**Comment by Frenzie** [comment]
Please be a lot more detailed. Which page shows what?

https://github.com/FreshRSS/Extensions/tree/be153ddb46297501aebe659151034e431140b14d/xExtension-ShareByEmail

**Reply by Frenzie** [reply]
Hold on, this is already answered in https://github.com/FreshRSS/FreshRSS/discussions/7098#discussioncomment-11554051

Please keep it there.

**Comment by math-GH** [comment]
I will improve the config slider for this extension
